### DISCUSSION TITLE
Your application cannot be reviewed until you fix the following:

### DISCUSSION THREAD

**Discussion by GalaxyBg** [community/community#64549] [GitHub Education]
### Hi everyone,

**I tried applying but after uploading student id card I just get the error** 

Your application cannot be reviewed until you fix the following:
You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name exactly as it appears in your academic affiliation document plus a short bio. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.

Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?

If your school does not provide academic email then you must enable use of your device camera to capture your image.

**I have put my name and a bio on my profile I have also logged out and then back in what am I supposed to do?**

**Comment by Kaysuto** [comment]
Hello,

It seems your GitHub profile does not exactly match the information on your student ID document.

Some things to check:

- Make sure the name on your GitHub profile matches the one on your ID document. No nicknames or variations.
- Check spelling, capitalization, accents etc. Everything must be identical.
- Add a short bio on your profile to complete it.
- Log out and log back into GitHub after making these changes so the profile updates.
- If your school does not provide an academic email, enable camera access in GitHub settings to add a picture of yourself.
- Make sure the ID image uploaded is good quality and readable.

Please let me know if you have any other questions! I'm happy to help diagnose the issue.

**Reply by GalaxyBg** [reply]
> enable camera access in GitHub settings to add a picture of yourself.

-Am I supposed to also have a picture of myself as my profile photo?

Also, student IDs in my country have text fields that you have to write by yourself(Including name, year, and basically almost everything) it is also not in English. Could this be the problem?
Besides that, I have done everything else you mentioned.

**Comment by Kaysuto** [comment]
It is recommended to apply for GitHub Education directly from the **GitHub mobile app** if you are having trouble validating your profile on the website.

Using the mobile app's webcam to add a profile picture can help get it added reliably.

If you still have trouble validating, I recommend checking out this discussion thread on the GitHub forums: https://github.com/orgs/community/discussions/52088

You'll find experiences and advice from other students who faced similar difficulties getting validated for GitHub Education access.

Feel free to let me know if those resources help you resolve your profile validation problem. Using the mobile app and consulting the forums are good first steps.

**Reply by GalaxyBg** [reply]
There is no such option on the mobile APP. I tried doing it through my phone's browser but I still get the same error. The discussion you listed doesn't seem to help either. 
**Edit:** I just tried using my school portal as proof and I now only get the error that I should take a picture and not upload one. The quality of the camera is very bad because it has to be taken from the site and even if I take the picture I still get the previous errors. Is there a way to contact support so they manually review it?
**Edit:** It now worked but it just returned me to the back to the page where you select an email and school and told me it is unable to verify it.
